orttraining/orttraining/python/training/__init__.py

```diff
@@ -3,13 +3,14 @@
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
 
-from onnxruntime.capi._pybind_state import TrainingParameters
-from onnxruntime.capi._pybind_state import PropagateCastOpsStrategy
+from onnxruntime.capi._pybind_state import GradientGraphBuilder, PropagateCastOpsStrategy, TrainingParameters
 from onnxruntime.capi.training.training_session import TrainingSession
 
+from .gradient_graph._gradient_graph_tools import export_gradient_graph
```

Comment:
Let's not add this here. Let uses explicitly call into the experimental module to call `export_gradient_graph`.